src/app/src/main/java/com/cmput301f21t09/budgetprojectname/views/activities/FollowRequestActivity.java

```diff
@@ -24,29 +28,57 @@
  */
 public class FollowRequestActivity extends AppCompatActivity {
 
+    // TODO: comment
     ArrayList<UserModel> followRequestDataList;
     ArrayAdapter<UserModel> followRequestAdapter;
 
+    /**
+     * Controller for fetching user details
+     */
+    private UserController userController;
+
+
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_follower_request);
 
+        // Get the Intent that started this activity and extract the user id string
+        Intent intent = getIntent();
+        String currentUserId = intent.getStringExtra("uid");
+        System.out.println("uid " + currentUserId);
+
         // ListView setup
         ListView followRequestList = findViewById(R.id.follow_request_listview);
 
         // Set up the list and send an empty list to the view
         followRequestDataList = new ArrayList<>();
-
-        // add a fake user to arrayList
-        UserModel fakeUser = new UserModel();
-        fakeUser.setFirstName("Fake");
-        fakeUser.setLastName("McFakerson");
-        fakeUser.setUsername("fk123");
-        followRequestDataList.add(fakeUser);
-
         followRequestAdapter = new FollowRequestCustomList(this, followRequestDataList);
         followRequestList.setAdapter(followRequestAdapter);
+        userController = new UserController();
```

Comment:
nit: Add a comment to this logic

Reply:
user controller is already defined above with a comment!